notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>In the second paragraph, you say "From plotting the two types of fluxes..." but only explain one of the types (KSPSAP_FLUX). Need to add a bit about SAP_FLUX.</p><p> </p><p>Also instead of "Such long timescale variation is intrinsic stellar variability" I would rephrase slightly: "This long timescale variability is caused by intrinsic stellar variations" or something similar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>